### PR TITLE
Rol branch

### PR DESCRIPTION
**Pull request by UrsuVasile**

### CODE REVIEW

File: src/main/java/Database/StockDao.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        Query query = session.createNamedQuery("total_stock");
+        long total = (long) query.getSingleResult();
+        closeSessionAndTransaction();
+        return total;
+    }
+
+    public List<Result> stockForEachProduct() {
+        openSessionAndTransaction();
+        Query query = session.createNamedQuery("stock_products");
+        List<Result> list = new LinkedList<>();
```
Comment (stefan2212) [inline src/main/java/Database/StockDao.java:22]: Linked list is slower at iterations. ArrayList is better here

Comment (stefan2212) [reply src/main/java/Database/StockDao.java:22]: List<Result> list = new LinkedList<>();
list = query.getResultList();

Can be replaced by = List<Result> list = query.getResultList(); you just need the interface

File: src/main/java/Service/Service.java [hunk 2]
```diff
@@ -1,13 +1,17 @@
 package Service;
```
Comment (stefan2212) [inline src/main/java/Service/Service.java:1]: You have a lot of Strings in this class.
As a good practice you can have a separate file where you can keep all those Strings in constants

**Review by stefan2212 — CHANGES_REQUESTED** [review]

File: src/main/java/Service/Service.java (hunk 2 above)
Comment (UrsuVasile) [reply src/main/java/Service/Service.java:1]: ok

File: src/main/java/Database/StockDao.java (hunk 1 above)
Comment (UrsuVasile) [reply src/main/java/Database/StockDao.java:22]: ok

**Review by stefan2212 — APPROVED** [review]

**Pull request merged**